lib/api.js

```diff
@@ -30,9 +33,9 @@ class API {
      *
      * @returns {undefined}
      */
-    constructor(config) {
+    constructor(name, config) {
```

Comment:
Mhh adding a name to the global API could be one way to enforce one common name for all modules, then the "name" provided to each module could become the "modulename" in the logs.
Why not :)